docs/setup/vagrant.rst

```diff
@@ -20,7 +22,9 @@ In the ``config`` directory, copy the file ``devconfig.json.sample`` to ``devcon
 Run ``vagrant plugin install vagrant-vbguest``
 If you are on Windows, also run ``vagrant plugin install vagrant-winnfsd``
 
-Run ``vagrant up && vagrant reload`` and wait while the development environment is set up. When asked to select a network interface for bridging, enter the number corresponding to one that is active. To automatically select this interface in the future, set the "network_interface" key in ``devconfig.json`` to the name of the interface you selected (e.g. ``"en0: Wi-Fi (AirPort)"``). There may be repeated warnings similar to "``Remote connection disconnect`` on the second ``vagrant up``. After several minutes they will stop. Once the provisioning process is complete, run ``vagrant ssh`` to log in to the development box.
+Run ``vagrant up && vagrant reload`` and wait while the development environment is set up. When you are asked to select a network interface for bridging, enter the number corresponding to one that is active. You may not need to select a network interface for bridging, which would allow you to skip this step. To automatically select this interface in the future, set the "network_interface" key in ``devconfig.json`` to the name of the interface you selected (e.g. ``"en0: Wi-Fi (AirPort)"``). There may be repeated warnings similar to ``Remote connection disconnect`` and ``Warning: connection aborted. Retrying...`` on the second ``vagrant up``. After several minutes they will stop. Once the provisioning process is complete, run ``vagrant ssh`` to log in to the development box.
```

Comment:
This is a little confusing. I would rather break the bridging instructions into a separate paragraph and say "If you are asked to select a network interface for bridging..."

Then the "There may be repeated warnings" section would go in another paragraph.